src/DonorProfiles/Helpers/SanitizeProfileData.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Give\DonorProfiles\Helpers;
+
+/**
+ * Normalize format of location type lists.
+ * @since 2.8.0
+ */
+class SanitizeProfileData {
+
+	/**
+	 * Sanitize int passed, and return 0 if somehting other than an integer was passed
+	 *
+	 * @param string $avatarId
+	 *
+	 * @return int
+	 *
+	 * @since 2.11.0
+	 */
+	public static function sanitizeInt( $avatarId ) {
+		return intval( $avatarId );
+	}
+
+	/**
+	 * Sanitize array of addresses passed
+	 *
+	 * @param array $addresses
+	 *
+	 * @return array
+	 *
+	 * @since 2.11.0
+	 */
+	public static function sanitizeAdditionalAddresses( $addresses ) {
+		foreach ( $addresses as $key => $value ) {
```

Comment:
Since this wraps `sanitizeAddress`, we could probably map over it, instead of using a loop.

```php
return array_map( $this->sanitizeAddress, $addresses );
```

Reply:
Done, ref: https://github.com/impress-org/givewp/pull/5486/commits/7c6f3e6f97dacf4928535707691bc239c5bc8903